sql/src/test/java/org/apache/druid/sql/calcite/CalciteWindowQueryTest.java

```diff
@@ -101,6 +102,7 @@ public CalciteWindowQueryTest(
   @SuppressWarnings("unchecked")
   public void windowQueryTest() throws IOException
   {
+    assumeTrue("These tests are only run in sqlCompatible mode!", NullHandling.sqlCompatible());
```

Comment:
Discussed with Zoltan and Eric - we're planning to support testing window function with default value mode as well since a lot of running deployments do use that feature. Zoltan will put up the change for testing that in a follow up, since that requires fixing another bug as well.